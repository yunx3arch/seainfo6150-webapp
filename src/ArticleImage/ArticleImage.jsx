import React from "react";
import PropTypes from "prop-types";
import styles from "./ArticleImage.module.css";

const ArticleImage =(props)=>{
    return(
        <img className={styles.img} src={props.url} alt = {props.title}/>
    );
};

ArticleImage.propTypes={
    url:PropTypes.string.isRequired,
    title:PropTypes.string.isRequired
}
export default ArticleImage;