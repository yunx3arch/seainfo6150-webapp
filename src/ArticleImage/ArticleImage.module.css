.img{
    border:10px solid red;
    border-radius: 50%;
    width:200px;
    height:100px;
    margin-top:10px;
    margin-bottom:10px;
    box-shadow:0 10px rgba(0, 0, 0, .3) ;
    
}
