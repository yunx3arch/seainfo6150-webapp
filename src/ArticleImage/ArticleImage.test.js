import { iteratee } from 'lodash';
import React from 'react';
import ArticleImage from './ArticleImage';

describe("articleImage test",()=>{
    it("renders correctly",()=>{
        const{container}=render(
            <ArticleImage
                title="The Statue of Liberty's torch heads to new museum"
                url="https://cdn.cnn.com/cnnnext/dam/assets/170321135754-airport-security-line-file-super-169.jpg"
            />
        );
        expect(container).toMatchSnapshot();
    })
})