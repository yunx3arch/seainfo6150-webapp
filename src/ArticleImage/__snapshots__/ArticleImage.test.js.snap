// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`articleImage test renders correctly 1`] = `
<div>
  <img
    alt="The Statue of Liberty's torch heads to new museum"
    class="image"
    src="https://cdn.cnn.com/cnnnext/dam/assets/170321135754-airport-security-line-file-super-169.jpg"
  />
</div>
`;
